lib/editor/item.dart

```diff
@@ -227,7 +227,11 @@ class _ListItemEditorState extends State<ListItemEditor> {
                     minLines: 1,
                     maxLines: 100,
                     cursorColor: Colors.black,
-                    decoration: const InputDecoration(border: InputBorder.none),
+                    cursorHeight: 16,
+                    decoration: InputDecoration(
+                      border: InputBorder.none,
+                      contentPadding: EdgeInsets.zero,
+                    ),
```

Comment:
where is this change coming from? This branch is supposed to be only for the math calculation errors, not any other fixes